infra/scripts/attest_persona_surface_coherence.py

```
@@ -0,0 +1,223 @@
+                f"registry persona `{key}` (ui id `{ui_id}`) is missing from the "
+                f"PERSONAS array in {DASHBOARD.relative_to(REPO_ROOT)} - it runs but "
+                f"no operator can see it"
+            )
+
+    # Reverse direction: a UI entry with no registry backing is a ghost.
+    for ui_id in sorted(ui_ids):
+        if ui_id not in REGISTRY_KEY_TO_UI_ID.values():
+            failures.append(
+                f"dashboard persona `{ui_id}` has no registry backing - it is shown to "
+                f"operators but nothing dispatches it"
+            )
```

Ruff (0.15.21)

[warning] Use `list.extend` to create a transformed list

(PERF401)